client/js/3d-brain.js

```diff
@@ -47,9 +83,18 @@ document.body.onmouseup = function() {
   --mouseDown;
 }
 
-var showStats = false;
+// Hard coded FA plot data length
+// TODO: Read this from an input file so that the user can change fidelity
+var faPlotLength = 100;
+
+var showStats = true;
```

Comment:
This variable controls the FPS stats shown on the top left? We probably want to set this back to false before merging, right?